docs/config.md

```diff
@@ -197,6 +201,47 @@ names:
   commonNamePattern: "{host}.buoyant.io"
 ```
 
+#### Load Balancer
+
+* *loadBalancer* -- Specifies a load balancer to use.  It must be an object
+containing keys:
+  * *kind* -- One of the supported load balancers.
+  * Any options specific to the load balancer.
+
+Current load balancers include:
+
+[p2c]: https://twitter.github.io/finagle/guide/Clients.html#power-of-two-choices-p2c-least-loaded
+[ewma]: https://twitter.github.io/finagle/guide/Clients.html#power-of-two-choices-p2c-peak-ewma
+[aperture]: https://twitter.github.io/finagle/guide/Clients.html#aperture-least-loaded
+[heap]: https://twitter.github.io/finagle/guide/Clients.html#heap-least-loaded
+
+##### [p2c][p2c]
+
+p2c supports the following options (see [here][p2c] for option semantics and defaults):
+
+* *maxEffort* -- Optional.
+
+##### [ewma][ewma]
+
+ewma supports the following options (see [here][ewma] for option semantics and defaults):
+
+* *maxEffort* -- Optional.
+* *decayTimeMs* -- Optional.
+
```

Comment:
per my understanding, p2c and ewma (and apperature) aren't mutually exclusive. I believe that EWMA is a costing strategy to feed into P2C, and apperature is a windowing strategy taht still uses P2C.  I think that "p2c" should actually be "queuedepth", here -- but I defer to someone who knows more about the details...